01_login.php

```diff
@@ -54,134 +55,132 @@
-				<a class="w3-bar-item"><b>logo</b> </a>
+				<a class="w3-bar-item"><b>Soundwave</b> </a>
+			</div>
+		</div>
 
-			</div></div>
 
+		<!-- background colour -->
+		<div class="w3-black">
+			<div class="w3-container w3-content w3-padding-64" style="max-width:100%">
 
-		<br>
-		<br>
-		<br>
-		<br>
-		<br>
 
+				<!-- Container (placeholder) -->
+				<div class="w3-content w3-container w3-padding-64">
 
-		<!-- logo image placing-->
-		<div class="w3-content w3-container w3-padding-64">
+					<div class="w3-row">
+						<div class="w3-col m6 w3-center w3-padding-large">
+							<img src="images/placeholder.jpg" class="w3-round w3-image w3-opacity" alt="placeholder" width="500" height="333">
+						</div>
 
-			<div class="w3-row">
-				<div class="w3-col m6 w3-center w3-padding-large">
 
-					<img src="images/placeholder.jpg" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Photo of Me" width="500" height="333">
-				</div>
+						<br>
 
+						<!-- brand name: soundwave -->
+						<div class="w3-col m6 w3-padding-large">
+							<h1>SOUNDWAVE</h1>
+							<form method = "post" id= "01_login">
 
-				<!-- brand + opening message -->
-				<div class="w3-col m6 w3-hide-small w3-padding-large">
-					<p>SOUNDWAVE</p>
-					<p>ↃꙄIUM ƎHT ⅃ƎƎꟻ</p>
+								<!-- login form -->
+								<div class="w3-row w3-section">
+									<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-user"> </i></div>
+									<div class="w3-rest">
+										<input for='login' class="w3-input w3-border" type = "username" name = "username" placeholder="Username" style="width:100%" required/>
+									</div>
+								</div>
 
+								<div class="w3-row w3-section">
+									<div class="w3-col" style="width:50px"><span class="w3-xxlarge material-icons">lock</span></div>
+									<div class="w3-rest">
+										<input for='login' class="w3-input w3-border" type = "password" name = "password" placeholder="Password" style="width:100%" required/>
+									</div>
+								</div>
 
-					<!-- login form -->
-					<form method = "post" id= "01_login"  class="w3-blue">
+								<p><button class="w3-button w3-section w3-white w3-ripple w3-hover-orange"  style="margin-left: 160px;"> Log in </button></p>
 
-						<div class="w3-row w3-section">
-							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-user"> </i></div>
-							<div class="w3-rest">
-								<input for='login' class="w3-input w3-border" type = "username" name = "username" placeholder="Username" style="width:100%" required/>
-							</div>
-						</div>
 
-						<div class="w3-row w3-section">
-							<div class="w3-col" style="width:50px"><span class="w3-xxlarge material-icons">lock</span></div>
-							<div class="w3-rest">
-								<input for='login' class="w3-input w3-border" type = "password" name = "password" placeholder="Password" style="width:100%" required/>
-							</div>
-						</div>
+								<hr>
```

Comment:
hr adds a line to separate the information from the login container but still keeping the information together.